resources/js/components/sections/welcomeJoin.vue

```diff
@@ -1,17 +1,39 @@
 <template>
-  <div class="welcomeSearch relative flex flex-wrap items-center justify-between">
-    <div class="container px-4 mx-auto my-12 flex flex-wrap items-center justify-between">
-      welcomeJoin
+  <main class="bg-gray-600 py-14 mb-10">
```

Comment:
Use of "main" for this section is incorrect. 

Use "article" instead

Reply:
I have changed the main to article. Whiles, I was implementing this change, I was wondering if using "section" will be better than "article" in this context.